CMakeLists.txt

```diff
@@ -5,12 +5,14 @@ cmake_minimum_required(VERSION 3.0)
 
 set(CMAKE_CONFIGURATION_TYPES Debug Optim CACHE TYPE INTERNAL FORCE)
 set(CMAKE_BUILD_TYPE "optim" CACHE STRING "Build Type: Debug or Optim")
+set(USE_VALGRIND_ALLOCATOR OFF CACHE BOOL "Use the valgrind-mode allocator")
 #set(ENABLE_FFTW OFF CACHE BOOL "Enable the FFTW FFT processor (GPL)")
 #set(ENABLE_NAYUKI_PORTABLE ON CACHE BOOL "Enable the Nayuki portable FFT processor (MIT)")
 #set(ENABLE_NAYUKI_AVX ON CACHE BOOL "Enable the Nayuki AVX assembly FFT processor (MIT)")
 #set(ENABLE_SPQLIOS_AVX ON CACHE BOOL "Enable the SPQLIOS AVX assembly FFT processor")
 #set(ENABLE_SPQLIOS_FMA ON CACHE BOOL "Enable the SPQLIOS FMA assembly FFT processor")
 set(ENABLE_TESTS ON CACHE BOOL "Build the tests (requires googletest)")
+set(ENABLE_BENCH ON CACHE BOOL "Build the benchmarks (requires google benchmark framework)")
```

Comment:
benchmarks in OFF mode by default
(we can provide the command line to compile the benchmarks in release mode with TFHE allocator)